src/main/java/pages/opportunity/OpportunityDetails.java

```diff
@@ -0,0 +1,114 @@
+package pages.opportunity;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.CacheLookup;
+import org.openqa.selenium.support.FindBy;
+import pages.basepages.DetailsBase;
+
+/**
+ * Created by Virginia Sanabria on 8/27/2015.
+ */
+public class OpportunityDetails extends DetailsBase {
+    @FindBy(id = "opp3_ileinner")
+    @CacheLookup
+    private WebElement opportunityName;
```

Comment:
Would be good add a suffix to know what kind of component it is, maybe it can be "Container" because it can be reference to a Div or Span or TD.